snuba/datasets/plans/storage_processing.py

```diff
@@ -73,13 +76,32 @@ def check_storage_readiness(storage: ReadableStorage) -> None:
             )
 
 
+def _is_downsampled_storage_key(storage_key: str) -> bool:
+    return bool(re.match(r"^StorageKey\.EAP_ITEMS_DOWNSAMPLE_\d+$", storage_key))
```

Comment:
I wonder if this conversion of `StorageKey` to a `str` is necessary. What if we do this:

```
def _is_downsampled_storage_key(storage_key: StorageKey) -> bool:
    return storage_key.value.startswith("eap_items_downsample_")

```